mulighetsrommet-api/src/main/resources/application-dev.yaml

```diff
@@ -96,7 +96,8 @@ app:
     synchronizeNorgEnheter:
       delayOfMinutes: 360 # Hver 6. time
     synchronizeNavAnsatte:
-      cronPattern: "0 0 6 * * *" # Hver morgen kl. 06:00
+      # cronPattern: "0 0 6 * * *" # Hver morgen kl. 06:00
+      cronPattern: "0 */5 * * * *" # Hvert 5. minutt
```

Comment:
Dette er for testing i dev før jeg merger. Kommer til å endre denne tilbake etter testing.